src/common/filesystembase.h

```diff
@@ -178,7 +178,7 @@ namespace FileSystem {
                 longStr.append("\\\\?");
             }
         } else {
-            longStr.append("\\\\?\\"); // prepend string by this four magic chars.
+            longStr.append(R"(\\?\)"); // prepend string by this four magic chars.
```

Comment:
I wonder why clang-tidy/clang-modernize changes this occurrence but not other occurrences like the one on line 178 (from `"\\\\?"` to `R"(\\?)"`.

Reply:
Good point, I'm not sure in fact... this construct isn't in the exclude list. I guess it has some sort of complexity threshold under which it doesn't convert. There doesn't seem to be a setting to control the sensibility: https://clang.llvm.org/extra/clang-tidy/checks/modernize-raw-string-literal.html

I'll look at completing this file. If you spotted other files which need to be completed feel free to point them out, I'll deal with them as well.